src/presentation/pages/Profile/MyCollection/MyCollection.module.scss

```diff
@@ -0,0 +1,104 @@
+@import 'app/styles/base/settings';
+
+.myCollection {
+  padding: 4rem;
+
+  &__collectionText {
+    margin: 1rem;
+    text-align: center;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__spinner {
+    display: flex;
+    justify-content: center;
+  }
+}
+
+.list {
+  &__title {
+    text-align: center;
+  }
+
+  &__item {
+    display: flex;
+    justify-content: center;
+
+    &Description {
+      height: 5rem;
+      overflow-y: hidden;
+    }
+
+    &Content {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      object-fit: cover;
+      background-color: $background-promotion;
+      margin: 1rem auto;
+      width: 99%;
+      text-align: left;
+      padding: 0 1rem;
+      height: 20rem;
+      position: relative;
+
+      &Hidden {
+        display: flex;
+        flex-direction: column;
+        position: absolute;
+        bottom: 0;
+        width: 100%;
+        padding: 1rem;
+        background: linear-gradient(0, #000000 14.58%, rgba(0, 0, 0, 0) 100%);
```

Comment:
Are u defining this kind of linear-gradient on constants ? Maybe if u are going to reuse it could be useful, if not forget it.